src/main/java/com/softserve/kickscooter/kickscootervehicle/management/converter/ScooterCreateDtoToEntity.java

```diff
@@ -0,0 +1,23 @@
+package com.softserve.kickscooter.kickscootervehicle.management.converter;
+
+
+import com.softserve.kickscooter.kickscootervehicle.management.dao.Scooter;
+import com.softserve.kickscooter.kickscootervehicle.management.dto.ScooterCreateDto;
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDateTime;
+
+
+@Component
+public class ScooterCreateDtoToEntity implements Converter<ScooterCreateDto, Scooter> {
+    @Override
+    public Scooter convert(ScooterCreateDto scooterCreateDto) {
+        Scooter scooter = new Scooter();
+        scooter.setModelName(scooterCreateDto.getModelName());
+        scooter.setSerialId(scooterCreateDto.getSerialId());
+        scooter.setRegisterDate(LocalDateTime.now());
```

Comment:
Nit: in real life, LocalDateTime is used very rarely. for fields like the timestamp of the creation, update, deletion it's more appropriate to use Instant. We've had a lot of problems because some companies used local time in their codebase.